common/views/components/Breadcrumb/Breadcrumb.tsx

```diff
@@ -41,11 +24,19 @@ const Breadcrumb: FunctionComponent<BreadcrumbItems> = ({
       .map(({ text, url, prefix }, i) => {
         const LinkOrSpanTag = url ? 'a' : 'span';
         return (
-          <ItemWrapper isFirst={i === 0} key={text} as={prefix ? 'b' : 'span'}>
+          <ItemWrapper key={text} as={prefix ? 'b' : 'span'}>
+            {i !== 0 && (
```

Comment:
this is a trivial nit, but since we know `i` is increasing, I feel this is a little clearer:

```suggestion
            {i > 0 && (
```